libs/util/git/src/git-client.service.ts

```diff
@@ -347,7 +350,7 @@ export class GitClientService {
       });
     }
 
-    return pullRequest.url;
+    return pullRequest?.url || "";
```

Comment:
As empty string is a valid value, I'd suggest to use the nullish coalescing operator instead.

```suggestion
    return pullRequest?.url ?? "";
```

Reply:
Done @overbit.